src/Microsoft.AspNet.Testing/xunit/OperatingSystems.cs

```diff
@@ -13,7 +13,6 @@ public enum OperatingSystems
         Win2008R2 = 1 << 1,
         Linux = 1 << 2,
         MacOSX = 1 << 3,
-
-        Win7And2008R2 = Win7 | Win2008R2,
+        AllWindows = 1 << 4
```

Comment:
I would rather prefer having just OSSkipConditionAttribute have just 'Linux, MacOSX and Windows' and write separate attributes for any specific version of Windows/Linux/Mac...